queries.sql

```diff
@@ -1,3 +1,26 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+
+/* Find all animals whose name ends in "mon". */
+SELECT * from animals WHERE like 'mon';
```

Comment:
👍 Good progress on your queries! However, there's a minor discrepancy in the query you've provided:

- [ ] In the screenshot you added to the PR, you used `%mom` as the pattern for animals' names. However, in the query you've included here, you've omitted the `%` sign. To ensure that the query works as intended, kindly update it to include the `%` sign before "mom" as shown in the screenshot.

By making this adjustment, you'll ensure that your query accurately reflects the screenshot and produces the correct results. Keep up the great work! 👍🔍

Reply:
Thanks alot! I have corrected it!